### PR TITLE
Excluding AKISerialNumber test on JDK8 until fix is merged

### PR DESCRIPTION
**Pull request by adamfarley**
This fix is in the dev branch, but is not yet merged into the master branch. This fix will likely be merged shortly after the April 2024 release.

### CODE REVIEW

**Review by smlambert — APPROVED** [review]

**Comment by smlambert** [comment]
> This fix is in the dev branch, but is not yet merged into the master branch. This fix will likely be merged shortly after the April 2024 release.

For awareness, typically we merge into master then [cherrypick](https://git-scm.com/docs/git-cherry-pick) the commit to the release branch.  There would be no delay in merging into master, as we do not code freeze AQAvit repos, since we tag and branch, development is not blocked/frozen during release periods.

**Comment by adamfarley** [comment]
In this instance then, should I cherry pick in the other direction and open a new PR so the commit ID is the same?

**Comment by smlambert** [comment]
> In this instance then, should I cherry pick in the other direction and open a new PR so the commit ID is the same?

No, in this instance, I have already merged your PR to the release branch.  We will proceed to merge this PR to master once it gets a second approval.

I mentioned it for awareness going forward for future PRs.  

And, we do not always want to / need to cherrypick commits to the release branch even though it makes sense to commit it to the master branch.

**Review by karianna — APPROVED** [review]

**Pull request merged**
